src/charts/heatmap/heatmap-legend.html

```diff
@@ -0,0 +1,6 @@
+<ul class="pf-heatmap-legend-container">
+    <li ng-repeat="item in legendItems" class="pf-heatmap-legend-items">
+        <span class="pf-legend-color-box" style="background: {{item.color}}"></span>
```

Comment:
This needs to be customer configurable (they should be able to set whatever legend color they want) so we can't really use a style here.